src/components/ExtensionMessage.sc.js

```diff
@@ -1,48 +1,38 @@
-import Box from "@mui/material/Box";
+import {
+  ModalWrapperGlobal,
+  CloseButtonGlobal,
+  ModalHeaderGlobal,
+  ModalBodyGlobal,
+  ModalFooterGlobal,
+} from "./ModalGlobalStyling.sc"; //file responsible for modal global styles settings
 import styled from "styled-components";
 
-const MyBox = styled(Box)`
-  position: absolute;
-  display: flex;
-  flex-direction: column;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  width: 40%;
-  max-width: 600px;
-  background-color: white;
-  border: 0 !important;
-  border-radius: 0.65em;
-  padding: 32px;
-  box-shadow: 0px 11px 15px -7px rgb(0 0 0 / 20%),
-    0px 24px 38px 3px rgb(0 0 0 / 14%), 0px 9px 46px 8px rgb(0 0 0 / 12%);
-  outline: none !important;
-
-  h1 {
-    font-size: 1.3em;
-    text-align: center;
-  }
-  a {
-    text-align: center;
-  }
-
-  .newAnnotation {
+const ExtensionMessageModalWrapper = styled(ModalWrapperGlobal)`
```

Comment:
In alignment with the other definitions below, this one should only be ModalWrapper I guess?

Reply:
I agree. I will update it in a minute